shared-helpers/src/types/backend-swagger.ts

```diff
@@ -2545,6 +2561,22 @@ export class ScriptRunnerService {
 
       configs.data = data
 
+      axios(configs, resolve, reject)
+    })
+  }
+  /**
+   * A script that removes empty race inputs
+   */
+  removeEmptyRaceInputs(options: IRequestOptions = {}): Promise<SuccessDTO> {
```

Comment:
This seems to be here because of a previous PR which didn't include an updated backend swagger file.